src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -35,12 +39,12 @@ secondary = true
 
 # Main dependencies
 [tool.poetry.dependencies]
-python = ">=3.7, <3.9"
+python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
-ansys-saf-glow = {version = "0.1.9", source = "solutions-private-pypi" }
-ansys-saf-portal = {version = "0.1.4", source = "solutions-private-pypi"}
-toml = {version = "*"}
-tox = {version = "*"} 
+ansys-saf-glow = {version = "^0.2", source = "solutions-private-pypi" }
+ansys-saf-portal = {version = "^0.1.6", source = "solutions-private-pypi"}
+toml = {version = ">=0.10.0,<=0.10.2"}
+tox = {version = ">=3.27.1,<=4.4.12"} 
```

Comment:
Please use caret to handle the bounds of the version.

Reply:
@iazehaf done